src/qt/optionsdialog.cpp

```diff
@@ -295,10 +295,22 @@ void OptionsDialog::on_resetButton_clicked()
 
 void OptionsDialog::on_openBitcoinConfButton_clicked()
 {
-    /* explain the purpose of the config file */
-    QMessageBox::information(this, tr("Configuration options"),
-        tr("The configuration file is used to specify advanced user options which override GUI settings. "
-           "Additionally, any command-line options will override this configuration file."));
+    QMessageBox config_msgbox;
+    config_msgbox.setIcon(QMessageBox::Information);
+    //: Window title text of pop-up box, which allows opening up configuration file.
+    config_msgbox.setWindowTitle(tr("Configuration options"));
+    /*: Self explanatory text about the priority order of instructions considered by client.
+        the order from high to low being: command-line -> configuration file -> GUI settings. */
```

Comment:
You cannot use the arrow in the translator comments; they show up as something else, see the output of `make -C src translate`:

```
<extracomment>Self explanatory text about the priority order of instructions considered by client. the order from high to low being: command-line -&gt; configuration file -&gt; GUI settings.</extracomment>
```

as you can see, the arrow's show up as `-&gt;`

```suggestion
        The order from high to low being: command-line, configuration file, GUI settings. */
```

Reply:
Addressed in 52aec5d.

Reply:
It is not specific to the combination of characters that to us mean an arrow: `->`. What's happening is that the `>` symbol gets converted to `&gt`.